pulsar-client-admin/pom.xml

```diff
@@ -95,6 +95,14 @@
       <groupId>com.google.code.gson</groupId>
       <artifactId>gson</artifactId>
     </dependency>
+
+    <!-- packages management service related dependencies (begin) -->
```

Comment:
AFAIK we are not used to adding this kind of "sections", please remove

Reply:
sure